src/Microsoft.ML.StandardLearners/Standard/LinearClassificationTrainer.cs

```diff
@@ -992,8 +985,7 @@ public StandardArrayDualsTable(int length)
                 _duals = new Float[length];
             }
 
-            public override Float this[long index]
-            {
+            public override Float this[long index] {
```

Comment:
(nit) inconsistent formatting here with the curly brace on the same line.

Reply:
Weird. My VS autoformatting had a strange setting about properties with newlines somehow, thanks Eric.